### PR TITLE
RA-1426 : Add link in General Actions menu to Enable Editing the Regi…

### PR DESCRIPTION
**Pull request by Ruhanga**
Ticket : https://issues.openmrs.org/browse/RA-1426

### CODE REVIEW

File: omod/src/main/resources/apps/refapp_registrationapp_app.json [hunk 1]
```diff
@@ -5,6 +5,16 @@
         "label": "Register Patient",
         "description": "Create a new Patient Record",
         "extensions": [
+            {
+                "id": "application.registrationapp.summary.editPatientLink",
+                "extensionPointId": "patientDashboard.overallActions",
+                "type": "link",
+                "label": "Summary/ Edit Patient",
```
Comment (mks-d) [inline omod/src/main/resources/apps/refapp_registrationapp_app.json]: This should be either of
* >`"Edit Registration Information"`
* >`"Edit Registration"`
* >`"Registration Summary"`

... and should be provided as an i18n message.

Comment (mks-d) [reply omod/src/main/resources/apps/refapp_registrationapp_app.json]: @mogoodrich any preference, or other suggestions?

**Comment by dkayiwa** [comment]
@Ruhanga is this ready to be merged?

File: omod/src/main/resources/apps/refapp_registrationapp_app.json (hunk 1 above)
Comment (mks-d) [reply omod/src/main/resources/apps/refapp_registrationapp_app.json]: @Ruhanga can you change this first:
```json
"label": "Edit Registration Information"
```

@dkayiwa after that we're good to merge I think ;-)

Comment (mks-d) [reply omod/src/main/resources/apps/refapp_registrationapp_app.json]: @Ruhanga also can you rely on a i18n message for this label, if possible?

Comment (Ruhanga) [reply omod/src/main/resources/apps/refapp_registrationapp_app.json]: I have worked on the comments @mks-d, except I'm thinking i18n should have been done with in the Reg App. This is trivial though.

**Pull request merged**